api-docs/client-interactions.md

```diff
@@ -0,0 +1,198 @@
+# Client Interactions
+
+The client interactions document direct encounters with the clients.
+
+Here is the list of the valid interaction types:
+
+- `clientInteractionCreated`,
+- `clientInteractionUpdated`,
+- `clientInteractionDeleted`,
+
+## Get a single Client Interaction
+
+### Request
+
+```http
+GET /api/clients/:id/interactions/:id
+```
+
+### Response
+
+```json
+{
```

Comment:
This object is missing a `serviceId`

Reply:
Like the "id": 1 but as "serviceId": 1, or is there a set of serivce ids already established?

Reply:
There is a set of service ids established that can be used